site/linkinator.config.json

```diff
@@ -1,4 +1,4 @@
 {
-  "skip": "^(https://.*github.com)",
+  "skip": "^(https://.*)",
```

Comment:
Probably better to just scan internal links (there are still 236 of them) as this will avoid flakes and some sites might block return 403 from github runners